### ISSUE TITLE
Opening message dialogue is choppy/laggy

### ISSUE THREAD

**Issue by luckycold** [krille-chan/fluffychat#927]
### Bug Description

I've noticed that at least on my phone (Pixel 8) when I open the dialogue box in any way (e.g. via reply or simple opening the message box) it opens at a different frame rate or something that's jarring from the rest of the app.

### Steps to Reproduce

1. Open any chat
2. Tap dialogue box
3. Witness animation of getting to the keyboard.

### Expected Behavior

Same smoothness of animations as rest of app.

### App Version

1.18.0

### Additional Platform Information

Pixel 8, Stock Android

### Additional Context

https://github.com/krille-chan/fluffychat/assets/26051327/ce6aca8a-e0dd-4045-a78f-f253d84261d0

**Comment by ReillyBrogan** [comment]
Yeah I've noticed this too on a Samsung S23 Ultra. I was trying different clients as an alternative to Element and Fluffychat seems to render noticeably poorly on my device. It does not feel smooth at all. This device has a 120hz screen and it feels like the app is rendering at sub-30hz.

**Comment by bachrc** [comment]
Yep, same here

**Comment by saihaze** [comment]
Is it a Fluffychat bug or an upstream bug?

**Comment by bachrc** [comment]
Well I don't have the problem with Element, Element X, schildi..

**Comment by imnotlxy** [comment]
https://github.com/krille-chan/fluffychat/pull/2081 should fix this
